examples/fuse_examples/tests/test_classification_cmmd.py

```diff
@@ -54,19 +54,19 @@ def run_cmmd(root: str) -> None:
                 "target": "classification",
                 "reset_cache": False,
                 "num_workers": 10,
```

Comment:
Why did you change cmmd?

Reply:
Forgot to mention but I played with it to speed-up the CI test (less data used and less epochs).
It's just a WA tho.. What do you think?

Reply:
Seems good.Lets keep it,
I think that the ccc command requesting cpu cores and specific GPU will fix it.